### PR TITLE
Adding spatialist

### PR DESCRIPTION
**Pull request by johntruckenbrodt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/spatialist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - libspatialite
+    - setuptools_scm
+
+test:
+  source_files:
+    - spatialist/tests
+  requires:
+    - pytest
+  imports:
+    - {{ name }}
```
Comment (synapticarbors) [inline recipes/spatialist/meta.yaml:43]: For the `imports`  down to the end of the `meta.yaml`, it preferable to not use jinja templating unnecessarily. It just makes it harder to look at the recipe and then figure out links before the recipe is rendered

File: recipes/spatialist/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  commands:
+    - $PYTHON -m pytest  # [not win]
```
Comment (synapticarbors) [inline recipes/spatialist/meta.yaml:45]: Can these commands be combined to just `{{ PYTHON }} -m pytest` or just `pytest`?

File: recipes/spatialist/meta.yaml (hunk 1 above)
Comment (johntruckenbrodt) [reply recipes/spatialist/meta.yaml]: Sure. Done. I'll remember that for future packages.

File: recipes/spatialist/meta.yaml (hunk 2 above)
Comment (johntruckenbrodt) [reply recipes/spatialist/meta.yaml]: `{{ PYTHON }} -m pytest` failed, but `pytest` works. Thanks.

**Comment by johntruckenbrodt** [comment]
Thanks a lot for the review @synapticarbors! It should be good to go now.

**Comment by synapticarbors** [comment]
@johntruckenbrodt -- Welcome to conda-forge and thank you for your contribution. If you run into any issues once the feedstock is created, don't hesitate to ping me or other members of conda-forge for help.

**Comment by johntruckenbrodt** [comment]
Awesome! Thanks a lot.

**Pull request merged**